src/nix/path-info.cc

```diff
@@ -45,6 +46,14 @@ struct CmdPathInfo : StorePathsCommand, MixJSON
             .description = "Show signatures.",
             .handler = {&showSigs, true},
         });
+
+        addFlag({
+            .longName = "query-substitutes",
```

Comment:
Maybe this should be named `--filter-substitutable` or something like that? Since it doesn't really query what  the substitutes are.